src/ui/map.js

```diff
@@ -101,7 +101,7 @@ type MapOptions = {
     locale?: Object
 };
 
-const defaultMinZoom = 0;
+const defaultMinZoom = -2;
```

Comment:
I think we should stick with 0 by default, and allow folks to explicitly pass in a negative `minZoom` if they really want to enable this.